magda-web-client/src/UI/DataPreviewVis.js

```diff
@@ -151,7 +151,8 @@ class DataPreviewVis extends Component {
             if (
                 previewData &&
```

Comment:
So what was the problem you had?

It'd be cool if it could be more like

```
if (previewData.meta.chartFields) {
   <Chart Tab>
}
if (previewData.meta.fields) {
   <Table Tab>
}
```

is it the `defaultSelectedIndex` that's the problem?